paasta_tools/mesos_tools.py

```diff
@@ -115,20 +115,32 @@ def get_current_tasks(job_id):
     return get_mesos_master().tasks(fltr=job_id, active_only=False)
 
 
+def is_task_running(task):
+    return task['state'] == 'TASK_RUNNING'
+
+
+def is_task_not_running(task):
+    return task['state'] != 'TASK_RUNNING'
+
+
+def filter_tasks(tasks, filter_function, **kwargs):
+    return [task for task in tasks if filter_function(task, **kwargs)]
```

Comment:
this is ``filter`` but with the arguments reorganised and the nomenclature around tasks? you could rewrite this with s/tasks/things & s/task/thing, and just have the ``filter`` function. I'd ditch this if I were you.